### ISSUE TITLE
Cannot install in angular v12.1.x because of peerDependency

### ISSUE THREAD

**Issue by Kilowhisky** [angular-architects/module-federation-plugin#79]
Hi,

I can't seem to install the plugin using `ng add` because there is a peer dependency conflict that doesn't make a lot of sense. 

https://github.com/angular-architects/module-federation-plugin/blob/606cec25bd5a931c9da79c9447ca05c7f369bdda/libs/mf-runtime/package.json#L6-L9

```
ℹ Using package manager: npm
✔ Found compatible package version: @angular-architects/module-federation@12.4.0.
✔ Package information loaded.

The package @angular-architects/module-federation@12.4.0 will be installed and executed.
Would you like to proceed? Yes
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @angular-architects/module-federation-runtime@12.4.0
npm WARN Found: @angular/common@undefined
npm WARN node_modules/@angular/common
npm WARN   @angular/common@"~12.1.0" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/common@">=12.4.0" from @angular-architects/module-federation-runtime@12.4.0
npm WARN node_modules/@angular-architects/module-federation-runtime
npm WARN   @angular-architects/module-federation-runtime@"^12.4.0" from @angular-architects/module-federation@12.4.0 
npm WARN   node_modules/@angular-architects/module-federation
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @angular-architects/module-federation-runtime@12.4.0
npm WARN Found: @angular/core@undefined
npm WARN node_modules/@angular/core
npm WARN   @angular/core@"~12.1.0" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/core@">=12.4.0" from @angular-architects/module-federation-runtime@12.4.0
npm WARN node_modules/@angular-architects/module-federation-runtime
npm WARN   @angular-architects/module-federation-runtime@"^12.4.0" from @angular-architects/module-federation@12.4.0 
npm WARN   node_modules/@angular-architects/module-federation
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @angular/common@>=12.4.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

There isn't even a beta version of `12.4.x` for angular at all. Is this a typo?

**Comment by gcollin** [comment]
To overcome that, I installed using `npm i @angular-architects/module-federation --legagy-peer-deps`
Then ng add ... works

**Comment by gcollin** [comment]
Then you have to change the dependency to 12.1.0 in the installed module's package.json

**Comment by manfredsteyer** [comment]
Thanks. Just fixed it with version 12.4.1
